### DISCUSSION TITLE
Default Wazuh Creds under Security Onion.

### DISCUSSION THREAD

**Discussion by APT256** [Security-Onion-Solutions/securityonion#8896] [Unsupported Versions] [answered]
Hello,
I just setup SO in my lab and configured several Windows lab servers with the wazuh agent. They appear to be talking fine, but I can't log into the wazuh web interface . I am going to: https://so.ip.address:55000 and I get prompted for a username and password. I have tried the creds I used to setup SO, but those don't work. The so-wazuh service is green and the "so-user list" command only shows the account that I used to setup SO with.

Can someone please tell me what the default creds are for wazuh under SO?

**Comment by reyesj2** [comment, marked as answer]
Looks like you're trying to access the wazuh api https://docs.securityonion.net/en/2.3/wazuh.html?highlight=wazuh#api

**Comment by APT256** [comment]
Ah Ha! That makes more sense. Thanks for the help!v/rJonOn Oct 11, 2022, at 16:02, Jorge Reyes ***@***.***> wrote:﻿
Looks like you're trying to access the wazuh api https://docs.securityonion.net/en/2.3/wazuh.html?highlight=wazuh#api

**Comment by APT256** [comment]
One more question... if I install wazuh separately, there is a web site for the wazuh manger. I don't see any mention of this in the SO docs.

**Reply by reyesj2** [reply]
All wazuh logs are ingested into elastic and available via the SOC https://docs.securityonion.net/en/2.3/wazuh.html#usage

The webpage you're referencing is when using Wazuh as an independent siem. Security Onion is it's own siem and incorporated the wazuh toolset for host visibility.

All your logs / alerts will be available in your soc dashboard. For your customization check out https://docs.securityonion.net/en/2.3/wazuh.html#tuning-rules
